src/api.js

```diff
@@ -0,0 +1,46 @@
+import axios from "axios";
+
+const fetchAllArticles = (topic) => {
+  if (topic === "all") {
```

Comment:
You can provide a second argument to the axios.get() function, which is an object that you can pass everything through, and if it's a truthy value it will add it to the url as a query. It's explained in the NCNotes entitled 'React Projects'